cmp/grid/Grid.scss

```diff
@@ -338,9 +341,32 @@
       color: var(--xh-text-color-muted);
     }
 
-    .xh-multifield-renderer-field,
+    .xh-multifield-renderer-field {
+      display: inline-block;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+
+    .xh-multifield-renderer-field-with-renderer {
```

Comment:
Checking with the team to see if we could rename this multifield-renderer and make it an implementation function specific to ZoneGrid.  That would give us the option for some better names, I think